pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -532,6 +538,20 @@ func (r *ReconcileLogStorage) Reconcile(ctx context.Context, request reconcile.R
 		dexCfg = render.NewDexRelyingPartyConfig(authentication, dexTLSSecret, dexSecret, r.clusterDomain)
 	}
 
+	// Cloud modifications
+	kbCm := &corev1.ConfigMap{}
+	key := types.NamespacedName{Name: "cloud-kibana-config", Namespace: rmeta.OperatorNamespace()}
```

Comment:
Is the `cloud-` prefix necessary? Seems like this is a generic feature of "setting custom kibana config"

Reply:
Certainly not necessary but I don't expect that this will be pushed into the main operator branch so I like keeping it obvious that this is a cloud operator only change and if anyone sees the config map the name will hopefully make that obvious.